regression_inputs/static_call_#XXX.json

```diff
@@ -0,0 +1,616 @@
+{
+  "Status": "running",
+  "Revision": "Istanbul",
+  "ReadOnly": true,
+  "Pc": 49,
+  "Gas": 1152921504606846976,
+  "GasRefund": 2475616058676252470,
+  "Code": "946ec9e280c51f5fb1c3b23c1424f8fde5e17503f23a34a33177323c130d7d1ee12a9fe0a7675716bcc3c7c963b9863f7df159ae9c6d9a0a309af431a16da1501d6c13306da6b198cce6c637b739285930ce53f0fbfb973fddf476f40dc2e95dc1eb36f18ba014",
+  "Stack": [
+    "94195109773ec5ba f3c9e5bc2e45b243 f5058fcff7a15984 6bfab812b045fd73",
```

Comment:
only the bottom 7 values are relevant for this call, the top of the stack can be removed.

Reply:
I removed this file, since it is not really covering an issue in any of the VM implementations but in the CT adaptors.